dcap/types/src/quote3.rs

```diff
@@ -66,9 +69,10 @@ impl Error {
 type Result<T> = ::core::result::Result<T, Error>;
 
 /// Quote version 3
-#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
```

Comment:
I'd suggest we
1. Derive `Ord, PartialOrd` on `ReportBody`, and 
2. Implement Eq, PartialEq, Ord, PartialOrd manually on this type, using only the bytes (no sense comparing the same data twice)